src/crimson/os/seastore/transaction_manager.h

```diff
@@ -363,6 +363,51 @@ class TransactionManager : public SegmentCleaner::ExtentCallbackInterface {
     });
   }
 
```

Comment:
Add an interface comment for this method as with the other methods in this file.  Make sure to note the restriction that paddr must be absolute.

Reply:
Also, add tests for this interface method to test_transaction_manager.cc.

Reply:
In addition to a basic test, please add a test validating the following scenario:

transaction 1:
alloc 16k extent at laddr 16k

transaction 2:
remove extent at laddr 16k
map existing 4k extent at laddr 16k
map existing 4k extent at laddr 20k
map existing 8k extent at laddr 24k
mutate 4k extent at laddr 20k

replay journal

The 4k extent at laddr 20k should be in state is_existing_pending(), I want to validate the mutating that extent generates a delta as expected.